### ISSUE TITLE
Accessibility > Contrast: Make it higher

### ISSUE THREAD

**Issue by pocc** [pocc/tshark.dev#18]
* [ ] __Background and foreground colors do not have a sufficient contrast ratio.__
Low-contrast text is difficult or impossible for many users to read. [Learn more](https://dequeuniversity.com/rules/axe/3.3/color-contrast).
    ```
    Failing Elements
    input#search-by
    a.highlight
    a.highlight
    a.highlight
    a.highlight
    a.highlight
    ...
     ```
